src/pkg/infrastructure/infrastructure.go

```diff
@@ -20,6 +22,7 @@ import (
 
 type Infra struct {
 	LocalDB            *gorm.DB
+	Cacher             *redis.Client
```

Comment:
Let's use regular map[] to store AccessTokens, and store RefreshTokens in LocalDB. Let's drop Redis, it's superfluous here.

Reply:
Its bad idea. How about launching this project in kubernetes more than one replica?

Reply:
We will not run this project in Kubernates because it is an MVP

Reply:
\+